src/IB/FEMechanicsBase.cpp

```diff
@@ -378,6 +423,111 @@ FEMechanicsBase::writeFEDataToRestartFile(const std::string& restart_dump_dirnam
 
 /////////////////////////////// PROTECTED ////////////////////////////////////
 
+void
+FEMechanicsBase::computeStaticPressure(PetscVector<double>& P_vec,
+                                       PetscVector<double>& X_vec,
+                                       const double /*data_time*/,
+                                       const unsigned int part)
+{
+    if (!d_static_pressure_part[part]) return;
+    const PressureProjectionType& proj_type = d_static_pressure_proj_type[part];
+    const VolumetricEnergyDerivativeFcn& U_prime_fcn = d_static_pressure_vol_energy_deriv_fcn[part];
+
+    // Extract the mesh.
+    EquationSystems& equation_systems = *d_equation_systems[part];
+    const MeshBase& mesh = equation_systems.get_mesh();
+    const BoundaryInfo& boundary_info = *mesh.boundary_info;
+    const unsigned int dim = mesh.mesh_dimension();
+
+    // Setup extra data needed to compute stresses/forces.
+
+    // Extract the FE systems and DOF maps, and setup the FE objects.
+    auto& P_system = equation_systems.get_system<ExplicitSystem>(PRESSURE_SYSTEM_NAME);
+    const DofMap& P_dof_map = P_system.get_dof_map();
+    FEDataManager::SystemDofMapCache& P_dof_map_cache = *d_fe_data[part]->getDofMapCache(PRESSURE_SYSTEM_NAME);
+    FEType P_fe_type = P_dof_map.variable_type(0);
+    std::vector<int> P_vars = { 0 };
+    std::vector<int> no_vars = {};
+    auto& X_system = equation_systems.get_system<ExplicitSystem>(COORDS_SYSTEM_NAME);
+    std::vector<int> X_vars(NDIM);
+    for (unsigned int d = 0; d < NDIM; ++d) X_vars[d] = d;
+
+    FEDataInterpolation fe(dim, d_fe_data[part]);
+    std::unique_ptr<QBase> qrule = QBase::build(QGAUSS, dim, FIFTH);
```

Comment:
This should probably be an input parameter (like the other quadrature orders for stress functions)

Reply:
In fact I think it may already be an input parameter (or at least have a variable associated with it).

Reply:
There is currently not a good way to customize the quadrature type or order except for stress functions. Let's address all of this in a separate PR. For now I will just make the behavior consistent with the other parts of this class.